price_feeder/tasks.py

```diff
@@ -326,15 +326,17 @@ def price_from_sources(self):
 
         return result
 
-    def post_price(self, price_no_precision, info_contracts, task=None, global_manager=None):
+    def post_price(self, price_no_precision, info_contracts, re_post, task=None, global_manager=None):
 
         # Not call until tx confirmed!
         pending_tx_receipt = pending_transaction_receipt(task)
-        if 'receipt' in pending_tx_receipt:
+        last_used_nonce = None
+        if 'receipt' in pending_tx_receipt and not re_post:
             if not pending_tx_receipt['receipt']['confirmed'] or pending_tx_receipt['receipt']['reverted']:
                 # Continue on pending status or reverted
                 return pending_tx_receipt
-
+        if 'receipt' in pending_tx_receipt and re_post:
+            last_used_nonce = web3.eth.getTransactionCount(network_manager.accounts[0].address)
```

Comment:
This assumes the only txs made by this account are price posts, right?

Reply:
yes, maybe it is better stores the nonce used for that tx